hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/Handler.java

```diff
@@ -54,4 +58,55 @@ public OzoneConfiguration createOzoneConfiguration() {
     return parent.createOzoneConfiguration();
   }
 
+  protected abstract OzoneAddress getAddress() throws OzoneClientException;
+
+  protected abstract void execute(OzoneClient client, OzoneAddress address)
+      throws IOException, OzoneClientException;
+
+  @Override
+  public Void call() throws Exception {
+    conf = createOzoneConfiguration();
+
+    OzoneAddress address = getAddress();
+    try (OzoneClient client = createClient(address)) {
+      if (isVerbose()) {
+        address.print(out());
+      }
```

Comment:
Just an idea, but aren't this few lines better placed inside createClient() to have this code clean from logging stuff?

Reply:
`createClient()` is overridden in `S3Handler`, so that would introduce some duplication.

Reply:
fair point